### PR TITLE
Typo fixes.

### PR DESCRIPTION
**Pull request by MisterSimple**
#3656.

### CODE REVIEW

**Comment by Rai99** [comment]
### **Is that a PR going into master**
gthaving everyone pick lucio _16bc6e089b9e808357194c98058b495b

**Comment by MisterSimple** [comment]
SHIT

**Comment by MisterSimple** [comment]
There, bomb defused. Ty R(AI).

**Comment by Rai99** [comment]
I would have caught you for #3635 too, but you fixed it before I said anything.

**Comment by MisterSimple** [comment]
Multi warned me about that one, can't believe I keep going to master

**Comment by jmwall24** [comment]
gonna start calling you MasterSimple

**Comment by Rai99** [comment]
Would that be a positive sign/name or a negative one?

**Comment by ThePotatoGuy** [comment]
Wait "everyday" is a real world, what's wrong with its usage here?

**Comment by Rai99** [comment]
I think he means to say that there's a difference in "everyday" and "every day". 

Everyday is used to described something that happened on a daily basis. ie. Everyday occurrence. 
Could be replaced with "regular" in most if not all cases and still make sense. 

Every day is used to describe each individual day itself. ie. Doing something every day. 
Could probably be replaced with "frequent"or "a lot" and still make sense I think, although not exactly the most ideal wording.

**Comment by ThePotatoGuy** [comment]
@jmwall24 assigning you as i have poor grammar

**Comment by MisterSimple** [comment]
The difference is the following:

"Everyday" is an adjective that means "commonplace" or "ordinary". As such, it should only be used alongside the noun it's describing. For example, in monika_justice, Monika says "everyday people", which is correct.

"Every day" is a phrase that means "each day". Example: "Every day, I imagine a future where I can be with you." Here, the adjective "every" modifies the noun" day".

An easy way to test which one you should use is to replace everyday/every day with "each day":

**Example:**

"I stayed optimistic every day, trying to find it."

vs.

"I stayed optimistic each day, trying to find it.

Since the sentence makes sense with "each day", then "every day" is correct.

**Example:**

"I mean, think about your everyday schedule."

vs.

"I mean, think about your each day schedule."

Here, the sentence doesn't make sense with "each day", so "everyday" is correct.

**Example:**

"I visit Monika every day, so it's an everyday activity."

vs.

"I visit Monika each day, so it's an each day activity."

"I visit Monika each day" is correct, but "so it's an each day activity" is incorrect.

File: Monika After Story/game/script-affection.rpy [hunk 1]
```diff
@@ -2372,7 +2372,7 @@ init python:
             filepath = "/My one and only love.txt"
             message = """\
 My dearest lover, friend, companion and owner of my heart...
-Everyday you make my dreams come true, a screen means nothing when you spend your time with me.
+Every day you make my dreams come true, a screen means nothing when you spend your time with me.
```
Comment (multimokia) [inline Monika After Story/game/script-affection.rpy]:
```suggestion
Every day, you make my dreams come true, a screen means nothing when you spend your time with me.
```

Comment (multimokia) [reply Monika After Story/game/script-affection.rpy]: One thing to make this a bit more correct.

Comment (MisterSimple) [reply Monika After Story/game/script-affection.rpy]: Approved

**Comment by monikLover** [comment]
Apporved

**Pull request merged**